compass/ocean/tests/global_ocean/init/initial_state.py

```diff
@@ -180,11 +180,9 @@ def run(self):
 
                 ds = ds.isel(Time=0)
 
-                if ('maxLevelCell' in ds) and ('minLevelCell' in ds):
+                if ('minLevelCell' in ds):
                     minLevelCell = ds.minLevelCell+1
-                    maxLevelCell = ds.maxLevelCell+1
```

Comment:
`maxLevelCell` did not need to be increased here, but I don't have an explanation for it.

Reply:
@cbegeman, maybe you figured this out but `maxLevelCell` is determined based on the bathymetry and the 1D vertical coordinate.  Because you've altered the 1D vertical coordinate already to have an index that is one greater than normal for a given depth, the index corresponding to the bathymetry will already be incremented.

I was also expecting you would need to do this at first (and also move the bathymetry somehow) but I now realize that all you had to do was alter the 1D vertical coordinate.